tileserver/__init__.py

```diff
@@ -234,9 +234,10 @@ def async_store(store, tile_data, coord, format):
 def async_update_tiles_of_interest(redis_cache_index, coord):
     """update tiles of interest set"""
     try:
-        redis_cache_index.index_coord(coord)
+        if coord.zoom <= 20:
+            redis_cache_index.index_coord(coord)
         if coord.zoom > 18:
-            coord_at_z18 = coord.zoomTo(18)
+            coord_at_z18 = coord.zoomTo(18).container()
```

Comment:
Ah, okay, so the only thing which matters is whether a tile coordinate is in the index or not, and the count is irrelevant. I was just thinking it would be weird to find the most popular tiles by sorting by the count, and z18 being overrepresented because of this.